tests/integration/golang/aim/project/get_project_test.go

```diff
@@ -0,0 +1,53 @@
+//go:build integration
+
+package run
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetProjectTestSuite struct {
+	suite.Suite
+	client          *helpers.HttpClient
+	projectFixtures fixtures.ProjectFixtures
+	project         response.GetProject
+}
+
+func TestGetProjectTestSuite(t *testing.T) {
+	suite.Run(t, new(GetProjectTestSuite))
+}
+
+func (s *GetProjectTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(helpers.GetServiceUri())
+
+	projectFixtures, err := fixtures.NewProjectFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.projectFixtures = *projectFixtures
+
+	project := projectFixtures.GetProject(context.Background())
+	s.project = *project
```

Comment:
why `project` here has type of `response.GetProject` instead of something related to `models`?

Reply:
the endpoint just returns a json response which has no model:
```        return c.JSON(fiber.Map{                                                                                                                     "name":              "FastTrackML",                                                                                                  "path":              database.DB.Dialector.Name(),                                                                                   "description":       "",                                                                                                             "telemetry_enabled": 0,                                                                                                      })```

Reply:
ahh, that's bad but ok. can you please just mark it with TODO so we can get back a bit later and think.

Reply:
adding TODO